docs/pages/database-access/guides/aws-cross-account.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: AWS Cross-Account Database Access
+description: How to connect AWS databases in external AWS accounts to Teleport.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  AWS cross-account database access is available starting from Teleport `13.0`.
+</Details>
+
+You can deploy the Teleport Database Service with AWS IAM credentials in one
+AWS account and use AWS IAM roles to grant Teleport access to databases in
+external AWS accounts.
```

Comment:
Can we mention as a prerequisite that in case of private VPC  connective between  teleport db agent and AWS DB instance needs to be configured ?

Reply:
I want to emphasize that the scope of this guide should be: 
- "how to setup AWS IAM rules and Teleport config to enable cross-account discovery/access"

I agree with you that this a pre-requisite, but I just wanted to emphasize what my intent is for this guide. There are many ways to configure VPC network security and connectivity, and it also depends on the type of database the user wants to connect to Teleport. I don't want to expand the scope of this guide to cover all the different ways to do this and all the details for each kind of database, but I agree we should remind them to do it. Our other guides, which they should consult for a specific kind of database, contain info about checking VPC connectivity as a troubleshooting step.

I've included this in a commit: 3d865f37a1a6be086f56c369042a27cb2c666f9b
```
## Prerequisites

(!docs/pages/includes/edition-prereqs-tabs.mdx!)

- A Teleport Database Service to connect your AWS database(s) to Teleport.
- AWS account(s) with permissions to modify and attach IAM policies.
- AWS VPC network configured to allow the Teleport Database Service to connect
  to your database(s).
```